src/hooks/useAuth.ts

```diff
@@ -61,6 +61,9 @@ const useAuth = () => {
   const logout = useCallback(() => {
     dispatch(profileClear())
     deactivate()
+    if (connectorsByName.walletconnect) {
```

Comment:
I think this returns you a constant string not the current connector

Reply:
One thing you can do is 

if (window.localStorage.getItem(ConnectorNames.WalletConnect)) {

Reply:
yeah it does, this is more of a check if its initialized or not.
I'm thinking of getting current connector but haven't looked yet how to do it without modifying UIKit. For now mostly made PR for preview link :)